add-to-cart/src/components/add-to-cart/src/add-to-cart.tsx

```diff
@@ -0,0 +1,111 @@
+import { Bindings, resultContext, initializeBindings } from "@coveo/atomic";
+import { Component, Host, h, Element, State, forceUpdate } from "@stencil/core";
+import { Result, Unsubscribe } from "@coveo/headless";
+
+interface CustomResult extends Result {
+  priceAndAvailability:
```

Comment:
This needs to be removed it is a client specific which render the component not generic